src/main/java/br/com/coddingSchool/dto/login/SubcategoryLoginDTO.java

```diff
@@ -0,0 +1,62 @@
+package br.com.coddingSchool.dto.login;
+
+import br.com.coddingSchool.model.Category;
+import br.com.coddingSchool.model.Course;
+import br.com.coddingSchool.model.Subcategory;
+
+import java.util.List;
+
+public class SubcategoryLoginDTO {
+
+    private Long id;
+    private String name;
+    private String code;
+    private int orderInSystem;
+    private boolean active;
+    private List<CourseLoginDTO> courses;
+    private Category category;
```

Comment:
Esse cara é usado apenas no Login? Me perdi no nome aqui, porque se for, acho que a projection resolve, se não muda o nome pra fazer mais sentido sabe